packages/@angular/cli/blueprints/ng/files/karma.conf.js

```diff
@@ -15,25 +15,14 @@ module.exports = function (config) {
     client:{
       clearContext: false // leave Jasmine Spec Runner output visible in browser
     },
-    files: [
```

Comment:
For projects using those, is this PR still working (even if it has the pre-PR performance hit)? Do we need to regenerate projects?